src/lqcontrol.jl

```diff
@@ -107,81 +83,103 @@ function update_values!(lq::LQ)
     # Set new state
     lq.P = new_P
     lq.d = new_d
-    return nothing
 end
 
-
 function stationary_values!(lq::LQ)
     # simplify notation
-    Q, R, A, B, C = lq.Q, lq.R, lq.A, lq.B, lq.C
+    Q, R, A, B, N, C = lq.Q, lq.R, lq.A, lq.B, lq.N, lq.C
 
     # solve Riccati equation, obtain P
-    A0, B0 = sqrt(lq.bet) .* A, sqrt(lq.bet) .* B
-    P = solve_discrete_riccati(A0, B0, R, Q)
+    A0, B0 = sqrt(lq.bet) * A, sqrt(lq.bet) * B
+    P = solve_discrete_riccati(A0, B0, R, Q, N)
 
     # Compute F
-    S1 = Q .+ lq.bet .* (B' * P * B)
-    S2 = lq.bet .* (B' * P * A)
-    F = S1 \ S2
+    s1 = Q + lq.bet * (B' * P * B)
+    s2 = lq.bet * (B' * P * A) + N
+    F = s1 \ s2
 
     # Compute d
-    d = lq.bet .* trace(P * C * C') / (1 - lq.bet)
+    d = lq.bet * trace(P * C * C') / (1 - lq.bet)
 
     # Bind states
     lq.P, lq.F, lq.d = P, F, d
-    nothing
 end
 
 function stationary_values(lq::LQ)
-    stationary_values!(lq)
-    return lq.P, lq.F, lq.d
+	_lq = LQ(
```

Comment:
My comment was not so much about the name as it was about the behavior of the function. The python version _does_ mutate the suppled instance of `LQ`, but we aren't here. I'm wondering which approach is better. 

Do we want `P, F, d` to be set to their stationary level, or should they stay at their initial values after this function runs?